evennia/commands/default/tests.py

```diff
@@ -239,10 +239,114 @@ def test_examine(self):
 
     def test_set_obj_alias(self):
         self.call(building.CmdSetObjAlias(), "Obj = TestObj1b", "Alias(es) for 'Obj(#4)' set to testobj1b.")
+        self.call(building.CmdSetObjAlias(), "", "Usage: @alias <obj> [= [alias[,alias ...]]]")
+
+    def test_set_obj_alias_caller_finds_nothing(self):
```

Comment:
Why not use the existing CmdTest mechanisms for these? It already mocks the msg method and allows you to compare if what is returned, so adding mocks to check for additional changes would be easier? At any rate, it appears to be a lot of boiler plate here - these tests could go into a new test class settings things up in its `setUp` method.